lib/finders/StoreFinder.js

```diff
@@ -40,7 +40,7 @@ var path = require('path'),
 	glob = require('glob');
 
 var DEFAULT_STORES_ROOT = 'catberry_stores',
-	STORES_GLOB = '**/*.js';
+	STORES_GLOB = '**/*.@(js|coffee)';
```

Comment:
We have to put this parameter to the [configuration](http://catberry.org/documentation#config) like it's done for componentsGlob parameter. Also, we should keep `STORES_GLOB` as `DEFAULT_STORES_GLOB` in this case.

IMO putting `.coffee` extension in Catberry's code isn't a good practice. Framework should know nothing about its plugins.